### PR TITLE
Cache instances of butler for subsequent requests

### PR DESCRIPTION
**Pull request by kennywlo**

### CODE REVIEW

File: python/lsst/dax/imgserv/locateImage.py [hunk 1]
```diff
@@ -80,203 +81,6 @@ def image_open_v1(W13db, config, logger=log):
...
-        self.metaservget = MetaservGet(self.conn, table, columns, logger)
-        try:
-            sql = "SET time_zone = '+0:00'"
-            self._log.info(sql)
-            self.conn.execute(sql)
-        except SQLAlchemyError as e:
-            self._log.error("Db engine error %s" % e)
-
-
-class ButlerGet:
```
Comment (ktlim) [inline python/lsst/dax/imgserv/locateImage.py]: I know it's fixed, but what was going on here?  A cut-and-paste issue?

File: python/lsst/dax/imgserv/getmetaserv.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+"""
+This module is used to fetch metadata based on astronomical parameters.
+
+@author: John Gates, SLAC
+@author: Brian Van Klaveren, SLAC
+@author: Kenny Lo, SLAC
+
+"""
+
+class MetaservGet:
```
Comment (ktlim) [inline python/lsst/dax/imgserv/getmetaserv.py]: Moving this to a separate file seems worthy of a separate commit.  Also see file naming comment above.

File: python/lsst/dax/imgserv/getbutler.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+@author: John Gates, SLAC
+@author: Brian Van Klaveren, SLAC
+@author: Kenny Lo, SLAC
+
+"""
+from flask import current_app, has_app_context
+import lsst.daf.persistence as dafPersist
+
+class ButlerGet:
```
Comment (ktlim) [inline python/lsst/dax/imgserv/getbutler.py]: The style guide says to name the `.py` file after the class that it contains.  https://developer.lsst.io/coding/python_style_guide.html#a-python-source-file-name-should-be-camelcase-with-leading-lowercase-and-ending-in-py

File: python/lsst/dax/imgserv/butlerGet.py [hunk 4]
```diff
@@ -0,0 +1,70 @@
+#
+# LSST Data Management System
+# Copyright 2015 LSST/AURA.
```
Comment (jgates108) [inline python/lsst/dax/imgserv/butlerGet.py]: Update copyright

File: python/lsst/dax/imgserv/metaservGet.py [hunk 5]
```diff
@@ -0,0 +1,98 @@
+#
+# LSST Data Management System
+# Copyright 2015 LSST/AURA.
```
Comment (jgates108) [inline python/lsst/dax/imgserv/metaservGet.py]: 2015-2017
